boards/seeed/xiao_nrf54l15/doc/index.rst

```diff
@@ -0,0 +1,104 @@
+
+- 128 MHz Arm® Cortex®-M33 processor
+- Scalable memory configurations up to 1.5 MB NVM and up to 256 KB RAM
+- Multiprotocol 2.4 GHz radio supporting Bluetooth Low Energy, 802.15.4-2020,
+    and 2.4 GHz proprietary modes (up to 4 Mbps)
+- Comprehensive set of peripherals including new Global RTC available in System OFF,
+    14-bit ADC, and high-speed serial interfaces
+- 128 MHz RISC-V coprocessor
+- Advanced security including TrustZone® isolation, tamper detection,
+    and cryptographic engine side-channel leakage protection
+
+
+For more information about the nRF54L15 SoC and XIAO nRF54L15 board, refer to these
+documents:
+
+- `nRF54L15 Website`_
+- `nRF54L15 Datasheet`_
+- `XIAO nRF54L15 Wiki`_
+
+Supported Features
+==================
+
+.. zephyr:board-supported-hw::
+
+Connections and IOs
+===================
+
+In the following table, the column **Name** contains Pin names. For example, P2_0
+means Pin number 0 on PORT2, as used in the board's datasheets and manuals.
+
++-------+-------------+------------------+
+| Name  | Function    | Usage            |
++=======+=============+==================+
+| P2_0  | GPIO        | LED0             |
++-------+-------------+------------------+
+| P1_9  | USART20_TX  | UART Console TX  |
++-------+-------------+------------------+
+| P1_8  | USART20_RX  | UART Console RX  |
++-------+-------------+------------------+
+
+
+Programming and Debugging
+*************************
+
+The XIAO nRF54L15 contains an SAMD11 with CMSIS-DAP, allowing flashing, debugging, logging, etc. over
+the USB port. Doing so requires a version of OpenOCD that includes support for the flash on the nRF54L15
+MCU. Until those changes are included in stock OpenOCD, the version bundled with Arduino can be
+used, or can be installed from the `OpenOCD Arduino Fork`_. When flashing, debugging, etc. you may
+need to include ``--openocd=/usr/local/bin/openocd
+--openocd-search=/usr/local/share/openocd/scripts/`` options to the command.
+
+Flashing
+========
+
+Connect the XIAO nRF54L15 board to your host computer using the USB port. A USB CDC ACM serial port
+should appear on the host, that can be used to view logs from the flashed application.
+
+Here is an example for the :zephyr:code-sample:`hello_world` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: xiao_nrf54l15
+   :goals: flash
+
+Open a serial terminal (minicom, putty, etc.) connecting to the UCB CDC ACM serial port.
+
+Reset the board and you should see the following message in the terminal:
+
+.. code-block:: console
+
+   Hello World! xiao_nrf54l15
+
+
+.. _XIAO nRF54L15 Wiki:
+   https://wiki.seeedstudio.com/getting_started_with_xiao_nrf54l15/
+
+.. _nRF54L15 Website:
+   https://www.nordicsemi.com/Products/nRF54L15
+
+.. _nRF54L15 Datasheet:
+   https://docs.nordicsemi.com/bundle/ps_nrf54L15/page/keyfeatures_html5.html
+
+.. _OpenOCD Arduino Fork:
+   https://github.com/facchinm/OpenOCD/tree/arduino-0.12.0-rtx5
```

Comment:
are you sure this is the URL you want/need? this doesn't look official at all? but maybe it's just the way it is and @facchinm's fork is where things happen, and not the `arduino` repo?